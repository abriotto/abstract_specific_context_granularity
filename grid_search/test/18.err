  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:01, 64.19it/s] 14%|##########################4                                                                                                                                                                | 14/99 [00:00<00:01, 63.21it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:00<00:01, 63.45it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:00<00:01, 66.88it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:00<00:00, 74.04it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:00<00:00, 65.21it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:00<00:00, 66.71it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:00<00:00, 65.76it/s] 72%|######################################################################################################################################1                                                    | 71/99 [00:01<00:00, 65.02it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:01<00:00, 57.27it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:01<00:00, 54.43it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:01<00:00, 59.40it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 55.97it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 61.46it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:00, 49.78it/s] 52%|#################################################################################################2                                                                                         | 13/25 [00:00<00:00, 60.45it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:00<00:00, 70.25it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 66.35it/s]
