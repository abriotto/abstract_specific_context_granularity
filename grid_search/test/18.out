# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7122822999954224, "acc": 0.47894877195358276, "length": 2.280683755874634, "mode": "train", "epoch": 1}
{"loss": 0.6870488524436951, "acc": 0.5395947694778442, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.7002597451210022, "acc": 0.496882826089859, "length": 1.0029642581939697, "mode": "train", "epoch": 2}
{"loss": 0.7074542045593262, "acc": 0.4318600594997406, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6959709525108337, "acc": 0.5073795318603516, "length": 1.0001169443130493, "mode": "train", "epoch": 3}
{"loss": 0.6921994686126709, "acc": 0.5211786031723022, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.696557343006134, "acc": 0.504920482635498, "length": 1.0001349449157715, "mode": "train", "epoch": 4}
{"loss": 0.6963688731193542, "acc": 0.47790056467056274, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6981563568115234, "acc": 0.5018450021743774, "length": 1.000082015991211, "mode": "train", "epoch": 5}
{"loss": 0.6970465183258057, "acc": 0.509208083152771, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6964380741119385, "acc": 0.4904828369617462, "length": 1.0015435218811035, "mode": "train", "epoch": 6}
{"loss": 0.6969959735870361, "acc": 0.4898711144924164, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6979091167449951, "acc": 0.4904666841030121, "length": 1.0000635385513306, "mode": "train", "epoch": 7}
{"loss": 0.69491046667099, "acc": 0.5009208917617798, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6955728530883789, "acc": 0.5064590573310852, "length": 1.0000947713851929, "mode": "train", "epoch": 8}
{"loss": 0.6944612860679626, "acc": 0.5184162259101868, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6955425143241882, "acc": 0.49631059169769287, "length": 1.000135064125061, "mode": "train", "epoch": 9}
{"loss": 0.691236138343811, "acc": 0.5248619318008423, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6944329142570496, "acc": 0.4953870475292206, "length": 1.0000507831573486, "mode": "train", "epoch": 10}
{"loss": 0.6986931562423706, "acc": 0.46593010425567627, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6940144300460815, "acc": 0.5052280426025391, "length": 1.0000340938568115, "mode": "train", "epoch": 11}
{"loss": 0.6973215341567993, "acc": 0.5027624368667603, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.693617045879364, "acc": 0.5187578201293945, "length": 1.0000560283660889, "mode": "train", "epoch": 12}
{"loss": 0.6941395998001099, "acc": 0.5239410996437073, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6934252977371216, "acc": 0.5003039836883545, "length": 1.0001128911972046, "mode": "train", "epoch": 13}
{"loss": 0.6921650767326355, "acc": 0.5165746212005615, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6963983178138733, "acc": 0.4907750189304352, "length": 1.0000255107879639, "mode": "train", "epoch": 14}
{"loss": 0.6977267265319824, "acc": 0.490791916847229, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.695108950138092, "acc": 0.5003075003623962, "length": 1.0000149011611938, "mode": "train", "epoch": 15}
{"loss": 0.6924771070480347, "acc": 0.4944751262664795, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6930244565010071, "acc": 0.5132226347923279, "length": 1.0000067949295044, "mode": "train", "epoch": 16}
{"loss": 0.6972126960754395, "acc": 0.5018416047096252, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6934143304824829, "acc": 0.5181516408920288, "length": 1.000380039215088, "mode": "train", "epoch": 17}
{"loss": 0.692983865737915, "acc": 0.5322283506393433, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6922846436500549, "acc": 0.5043094158172607, "length": 1.0008734464645386, "mode": "train", "epoch": 18}
{"loss": 0.6981968879699707, "acc": 0.47790059447288513, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6923453211784363, "acc": 0.5177968144416809, "length": 1.0024440288543701, "mode": "train", "epoch": 19}
{"loss": 0.6939165592193604, "acc": 0.4963168203830719, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.6921805739402771, "acc": 0.5212188363075256, "length": 1.0000941753387451, "mode": "train", "epoch": 20}
{"loss": 0.6962764263153076, "acc": 0.4825046956539154, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6922212243080139, "acc": 0.5261382460594177, "length": 1.000096321105957, "mode": "train", "epoch": 21}
{"loss": 0.6968170404434204, "acc": 0.5036832690238953, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.6907850503921509, "acc": 0.5205954909324646, "length": 1.0025042295455933, "mode": "train", "epoch": 22}
{"loss": 0.6938570737838745, "acc": 0.5055249333381653, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.6924164891242981, "acc": 0.5227811932563782, "length": 1.001276969909668, "mode": "train", "epoch": 23}
{"loss": 0.6998686790466309, "acc": 0.4668509066104889, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6943851709365845, "acc": 0.5172204971313477, "length": 1.0000602006912231, "mode": "train", "epoch": 24}
{"loss": 0.6905604600906372, "acc": 0.529465913772583, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.6918358206748962, "acc": 0.5242927670478821, "length": 1.0000004768371582, "mode": "train", "epoch": 25}
{"loss": 0.6981613039970398, "acc": 0.48895028233528137, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.6906423568725586, "acc": 0.5138375759124756, "length": 1.0000017881393433, "mode": "train", "epoch": 26}
{"loss": 0.6937717795372009, "acc": 0.50184166431427, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6905742883682251, "acc": 0.5199876427650452, "length": 1.000005841255188, "mode": "train", "epoch": 27}
{"loss": 0.6980313062667847, "acc": 0.46869245171546936, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6896796822547913, "acc": 0.5301353335380554, "length": 1.000001072883606, "mode": "train", "epoch": 28}
{"loss": 0.6964351534843445, "acc": 0.49815836548805237, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.689609169960022, "acc": 0.5292239785194397, "length": 1.001359224319458, "mode": "train", "epoch": 29}
{"loss": 0.6961173415184021, "acc": 0.483425498008728, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.6855678558349609, "acc": 0.5522046089172363, "length": 1.0866655111312866, "mode": "train", "epoch": 30}
{"loss": 0.678539514541626, "acc": 0.5363048911094666, "length": 1.4143646955490112, "mode": "test", "epoch": 30}
{"loss": 0.6426074504852295, "acc": 0.6391900181770325, "length": 1.979353904724121, "mode": "train", "epoch": 31}
{"loss": 0.6608414053916931, "acc": 0.6133747696876526, "length": 2.2486188411712646, "mode": "test", "epoch": 31}
{"loss": 0.6141104698181152, "acc": 0.6644933223724365, "length": 2.2504332065582275, "mode": "train", "epoch": 32}
{"loss": 0.6380862593650818, "acc": 0.6384257674217224, "length": 2.408839702606201, "mode": "test", "epoch": 32}
{"loss": 0.5882375240325928, "acc": 0.68915855884552, "length": 2.2592110633850098, "mode": "train", "epoch": 33}
{"loss": 0.610599935054779, "acc": 0.6849321126937866, "length": 2.110497236251831, "mode": "test", "epoch": 33}
{"loss": 0.5758025050163269, "acc": 0.7032434940338135, "length": 2.2390828132629395, "mode": "train", "epoch": 34}
{"loss": 0.6044111251831055, "acc": 0.6553347706794739, "length": 2.309392213821411, "mode": "test", "epoch": 34}
{"loss": 0.5722798705101013, "acc": 0.694849967956543, "length": 2.3222365379333496, "mode": "train", "epoch": 35}
{"loss": 0.592280387878418, "acc": 0.6631314158439636, "length": 2.657458543777466, "mode": "test", "epoch": 35}
{"loss": 0.5621708035469055, "acc": 0.706425130367279, "length": 2.7147579193115234, "mode": "train", "epoch": 36}
{"loss": 0.5791791677474976, "acc": 0.6969465613365173, "length": 2.790055274963379, "mode": "test", "epoch": 36}
{"loss": 0.5570147633552551, "acc": 0.7127984762191772, "length": 2.6218085289001465, "mode": "train", "epoch": 37}
{"loss": 0.5708786249160767, "acc": 0.728558361530304, "length": 2.5248618125915527, "mode": "test", "epoch": 37}
{"loss": 0.5357208847999573, "acc": 0.7378559112548828, "length": 2.6251842975616455, "mode": "train", "epoch": 38}
{"loss": 0.5552117824554443, "acc": 0.7382309436798096, "length": 2.8397789001464844, "mode": "test", "epoch": 38}
{"loss": 0.5270641446113586, "acc": 0.7493627071380615, "length": 2.824152946472168, "mode": "train", "epoch": 39}
{"loss": 0.5453524589538574, "acc": 0.7466415762901306, "length": 2.823204517364502, "mode": "test", "epoch": 39}
{"loss": 0.5211570262908936, "acc": 0.7490361928939819, "length": 2.8869965076446533, "mode": "train", "epoch": 40}
{"loss": 0.5501159429550171, "acc": 0.7070220708847046, "length": 2.756906032562256, "mode": "test", "epoch": 40}
{"loss": 0.5200144648551941, "acc": 0.7455713748931885, "length": 2.856053352355957, "mode": "train", "epoch": 41}
{"loss": 0.533014178276062, "acc": 0.7625254988670349, "length": 2.8397789001464844, "mode": "test", "epoch": 41}
{"loss": 0.5148699283599854, "acc": 0.7552841305732727, "length": 2.8357930183410645, "mode": "train", "epoch": 42}
{"loss": 0.529029369354248, "acc": 0.7599398493766785, "length": 2.988950252532959, "mode": "test", "epoch": 42}
{"loss": 0.510249137878418, "acc": 0.7588048577308655, "length": 2.9454050064086914, "mode": "train", "epoch": 43}
{"loss": 0.5290451049804688, "acc": 0.7325224280357361, "length": 3.022099494934082, "mode": "test", "epoch": 43}
{"loss": 0.5065022110939026, "acc": 0.7542586326599121, "length": 2.962074041366577, "mode": "train", "epoch": 44}
{"loss": 0.5139012336730957, "acc": 0.7670870423316956, "length": 3.0055248737335205, "mode": "test", "epoch": 44}
{"loss": 0.501343846321106, "acc": 0.7618380784988403, "length": 3.0821752548217773, "mode": "train", "epoch": 45}
{"loss": 0.5144268870353699, "acc": 0.7556207776069641, "length": 3.187845230102539, "mode": "test", "epoch": 45}
{"loss": 0.49841246008872986, "acc": 0.7605708837509155, "length": 3.0042619705200195, "mode": "train", "epoch": 46}
{"loss": 0.5163824558258057, "acc": 0.7669323086738586, "length": 3.055248737335205, "mode": "test", "epoch": 46}
{"loss": 0.4886990189552307, "acc": 0.7725843191146851, "length": 3.1067891120910645, "mode": "train", "epoch": 47}
{"loss": 0.5165325403213501, "acc": 0.7080674767494202, "length": 3.121546983718872, "mode": "test", "epoch": 47}
{"loss": 0.480135977268219, "acc": 0.7724553346633911, "length": 3.2391610145568848, "mode": "train", "epoch": 48}
{"loss": 0.4844430685043335, "acc": 0.7704758048057556, "length": 3.3370165824890137, "mode": "test", "epoch": 48}
{"loss": 0.46935024857521057, "acc": 0.7753524780273438, "length": 3.2647886276245117, "mode": "train", "epoch": 49}
{"loss": 0.5070803165435791, "acc": 0.7643493413925171, "length": 3.2375690937042236, "mode": "test", "epoch": 49}
{"loss": 0.46247127652168274, "acc": 0.7865013480186462, "length": 3.2548022270202637, "mode": "train", "epoch": 50}
{"loss": 0.46353405714035034, "acc": 0.797594428062439, "length": 3.220994472503662, "mode": "test", "epoch": 50}
{"loss": 0.4726499617099762, "acc": 0.787470281124115, "length": 3.368116617202759, "mode": "train", "epoch": 51}
{"loss": 0.4805595874786377, "acc": 0.7669102549552917, "length": 3.254143714904785, "mode": "test", "epoch": 51}
{"loss": 0.4608410596847534, "acc": 0.7852152585983276, "length": 3.248487949371338, "mode": "train", "epoch": 52}
{"loss": 0.4862709641456604, "acc": 0.7406768798828125, "length": 3.3701658248901367, "mode": "test", "epoch": 52}
{"loss": 0.4439411461353302, "acc": 0.798494815826416, "length": 3.392167568206787, "mode": "train", "epoch": 53}
{"loss": 0.48137959837913513, "acc": 0.7775928378105164, "length": 3.2707183361053467, "mode": "test", "epoch": 53}
{"loss": 0.4403354525566101, "acc": 0.7931196093559265, "length": 3.2900359630584717, "mode": "train", "epoch": 54}
{"loss": 0.48376578092575073, "acc": 0.7759991884231567, "length": 3.088397741317749, "mode": "test", "epoch": 54}
{"loss": 0.4316409230232239, "acc": 0.8023229241371155, "length": 3.260256767272949, "mode": "train", "epoch": 55}
{"loss": 0.4570309519767761, "acc": 0.8039688467979431, "length": 3.3867404460906982, "mode": "test", "epoch": 55}
{"loss": 0.4172656834125519, "acc": 0.8203418850898743, "length": 3.585437536239624, "mode": "train", "epoch": 56}
{"loss": 0.460428923368454, "acc": 0.7929642796516418, "length": 3.7513811588287354, "mode": "test", "epoch": 56}
{"loss": 0.4000350534915924, "acc": 0.8371085524559021, "length": 3.890700578689575, "mode": "train", "epoch": 57}
{"loss": 0.4584566056728363, "acc": 0.7882137298583984, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.40500620007514954, "acc": 0.830908477306366, "length": 3.9878811836242676, "mode": "train", "epoch": 58}
{"loss": 0.47032827138900757, "acc": 0.7817679643630981, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3923453092575073, "acc": 0.8330239653587341, "length": 3.9825668334960938, "mode": "train", "epoch": 59}
{"loss": 0.4561786651611328, "acc": 0.8093922734260559, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.38536515831947327, "acc": 0.8378911018371582, "length": 3.9963486194610596, "mode": "train", "epoch": 60}
{"loss": 0.4453660845756531, "acc": 0.8121547102928162, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.36783790588378906, "acc": 0.856196403503418, "length": 3.9894790649414062, "mode": "train", "epoch": 61}
{"loss": 0.42273443937301636, "acc": 0.8213629126548767, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.35781943798065186, "acc": 0.8566318154335022, "length": 3.991788387298584, "mode": "train", "epoch": 62}
{"loss": 0.4047313630580902, "acc": 0.8333333730697632, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.34913524985313416, "acc": 0.8578839898109436, "length": 3.9915425777435303, "mode": "train", "epoch": 63}
{"loss": 0.4005144536495209, "acc": 0.8425414562225342, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.32687675952911377, "acc": 0.8717012405395508, "length": 3.994589328765869, "mode": "train", "epoch": 64}
{"loss": 0.40505295991897583, "acc": 0.8453039526939392, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.3359004557132721, "acc": 0.861306369304657, "length": 3.9965596199035645, "mode": "train", "epoch": 65}
{"loss": 0.4145636558532715, "acc": 0.8241252303123474, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.3178994953632355, "acc": 0.8714313507080078, "length": 3.9985198974609375, "mode": "train", "epoch": 66}
{"loss": 0.3804450035095215, "acc": 0.8489872217178345, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.31897398829460144, "acc": 0.8692052364349365, "length": 3.9949073791503906, "mode": "train", "epoch": 67}
{"loss": 0.37926673889160156, "acc": 0.8462247252464294, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.31115543842315674, "acc": 0.87791907787323, "length": 3.999434232711792, "mode": "train", "epoch": 68}
{"loss": 0.39447081089019775, "acc": 0.8379373550415039, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.306948184967041, "acc": 0.8754411935806274, "length": 3.9994077682495117, "mode": "train", "epoch": 69}
{"loss": 0.3704856336116791, "acc": 0.841620683670044, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.29823750257492065, "acc": 0.8779210448265076, "length": 3.999847650527954, "mode": "train", "epoch": 70}
{"loss": 0.3861442804336548, "acc": 0.842541515827179, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.294399619102478, "acc": 0.8837611079216003, "length": 3.999722480773926, "mode": "train", "epoch": 71}
{"loss": 0.3735201358795166, "acc": 0.8508288264274597, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.2824651598930359, "acc": 0.8868368864059448, "length": 3.999917507171631, "mode": "train", "epoch": 72}
{"loss": 0.3957473635673523, "acc": 0.8489870429039001, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.29194891452789307, "acc": 0.878225564956665, "length": 3.9997310638427734, "mode": "train", "epoch": 73}
{"loss": 0.3905118703842163, "acc": 0.8388582468032837, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.2943664789199829, "acc": 0.883760929107666, "length": 3.9998505115509033, "mode": "train", "epoch": 74}
{"loss": 0.3601621389389038, "acc": 0.861878514289856, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.29729893803596497, "acc": 0.883456289768219, "length": 3.9999818801879883, "mode": "train", "epoch": 75}
{"loss": 0.35877904295921326, "acc": 0.8489871621131897, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.29218968749046326, "acc": 0.8852802515029907, "length": 3.9991414546966553, "mode": "train", "epoch": 76}
{"loss": 0.4188438355922699, "acc": 0.8232043385505676, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.29069873690605164, "acc": 0.8816110491752625, "length": 3.9998879432678223, "mode": "train", "epoch": 77}
{"loss": 0.37996071577072144, "acc": 0.8480663895606995, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.2912014126777649, "acc": 0.888374924659729, "length": 3.9998562335968018, "mode": "train", "epoch": 78}
{"loss": 0.37747031450271606, "acc": 0.842541515827179, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.27628278732299805, "acc": 0.8905339241027832, "length": 3.9990782737731934, "mode": "train", "epoch": 79}
{"loss": 0.36263617873191833, "acc": 0.8535909652709961, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.2713435888290405, "acc": 0.8985235095024109, "length": 3.99997878074646, "mode": "train", "epoch": 80}
{"loss": 0.35144945979118347, "acc": 0.8591160178184509, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.2695869505405426, "acc": 0.8960638642311096, "length": 3.9999938011169434, "mode": "train", "epoch": 81}
{"loss": 0.3541470468044281, "acc": 0.8517495393753052, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.2654445469379425, "acc": 0.8991377949714661, "length": 3.9999613761901855, "mode": "train", "epoch": 82}
{"loss": 0.3489009141921997, "acc": 0.8655617237091064, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.2587704360485077, "acc": 0.8972936272621155, "length": 3.99997878074646, "mode": "train", "epoch": 83}
{"loss": 0.35229748487472534, "acc": 0.8517496585845947, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.2573985159397125, "acc": 0.8975811004638672, "length": 3.999237298965454, "mode": "train", "epoch": 84}
{"loss": 0.3501608371734619, "acc": 0.8646409511566162, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.264151930809021, "acc": 0.8932952284812927, "length": 3.9999735355377197, "mode": "train", "epoch": 85}
{"loss": 0.3844836354255676, "acc": 0.8517496585845947, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.27300843596458435, "acc": 0.8945432901382446, "length": 3.9986157417297363, "mode": "train", "epoch": 86}
{"loss": 0.39767056703567505, "acc": 0.8618784546852112, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.2621448338031769, "acc": 0.8976003527641296, "length": 3.999866247177124, "mode": "train", "epoch": 87}
{"loss": 0.421354740858078, "acc": 0.8351749777793884, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.253473699092865, "acc": 0.9000588655471802, "length": 3.99989914894104, "mode": "train", "epoch": 88}
{"loss": 0.4025599956512451, "acc": 0.8453039526939392, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.2742269039154053, "acc": 0.8923739790916443, "length": 3.9999797344207764, "mode": "train", "epoch": 89}
{"loss": 0.3793580234050751, "acc": 0.8545119762420654, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.2646375894546509, "acc": 0.8945262432098389, "length": 3.9999923706054688, "mode": "train", "epoch": 90}
{"loss": 0.39887404441833496, "acc": 0.8508288264274597, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.25120609998703003, "acc": 0.9003690481185913, "length": 3.9999899864196777, "mode": "train", "epoch": 91}
{"loss": 0.3697889745235443, "acc": 0.8535911440849304, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.24690666794776917, "acc": 0.9040581583976746, "length": 3.9999730587005615, "mode": "train", "epoch": 92}
{"loss": 0.36602696776390076, "acc": 0.852670431137085, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.24995820224285126, "acc": 0.9015981554985046, "length": 3.9999616146087646, "mode": "train", "epoch": 93}
{"loss": 0.36509546637535095, "acc": 0.8581952452659607, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.25156915187835693, "acc": 0.9037505388259888, "length": 3.9999735355377197, "mode": "train", "epoch": 94}
{"loss": 0.37122124433517456, "acc": 0.8581952452659607, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.25451478362083435, "acc": 0.9040582776069641, "length": 3.9999802112579346, "mode": "train", "epoch": 95}
{"loss": 0.4346901774406433, "acc": 0.8351749777793884, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.23503610491752625, "acc": 0.9065189957618713, "length": 3.999969720840454, "mode": "train", "epoch": 96}
{"loss": 0.39009571075439453, "acc": 0.860957682132721, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.2458907812833786, "acc": 0.9034441113471985, "length": 3.999983310699463, "mode": "train", "epoch": 97}
{"loss": 0.3820766806602478, "acc": 0.8563537001609802, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.24710296094417572, "acc": 0.9031366109848022, "length": 3.9999797344207764, "mode": "train", "epoch": 98}
{"loss": 0.381640762090683, "acc": 0.8664824962615967, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.25835755467414856, "acc": 0.8979027271270752, "length": 3.9988090991973877, "mode": "train", "epoch": 99}
{"loss": 0.4041745364665985, "acc": 0.8535911440849304, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.2562342584133148, "acc": 0.8997550010681152, "length": 3.9999234676361084, "mode": "train", "epoch": 100}
{"loss": 0.3737766146659851, "acc": 0.8683242201805115, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.777777910232544
